src/content/troubleshooting/faq/timestamps.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: "../../../layouts/FAQLayout.astro"
+sidebar: { hide: true }
+title: How are timestamps displayed in Chromatic?
+section: usage
+---
+
+# How are timestamps displayed in Chromatic?
+
+While our capture and webapp servers are set to UTC, all timestamps in Chromatic are displayed in your system's local time.
```

Comment:
To clarify, Chromatic's web app will show local time in Canvas view. But snapshots will show server time, right? If so, might be worth calling that out.

Reply:
good point — this concerns timestamps in the web app (e.g., build and billing screens), not so much canvas/snapshot.

Reply:
Okay that makes a lot of sense. I was confused for some reason. My first thought was timestamps in test code 🤦🏽‍♂️ 

Maybe add the "(e.g., build and billing screens)" to the text?

Reply:
for sure, thanks varun!